src/utils/topup.ts

```diff
@@ -70,6 +70,11 @@ export const TopupProviders: { [providerName: string]: TopupProvider } = {
         label: "SOL",
         symbol: "sol",
       },
+      {
+        value: "USDC (SOL)",
```

Comment:
USC sol require token account address for moonpay topup
User may also do not have usdc token account which we might need new flow for user to create usdc token account
Suggest disable usdc topup for now